Gemfile

```diff
@@ -17,12 +17,13 @@ gem 'bootstrap_form'
 gem 'sidekiq'
 #gem 'sentry-raven'
 
+gem 'fog'
+gem 'fog-aws'
```

Comment:
This gem does not work correctly, in my opinion, so its more likely only fog itself.